src/app/tests/suites/certification/Test_TC_SC_5_1.yaml

```diff
@@ -13,257 +13,226 @@
-          [1650610759.578251][2739:2744] CHIP:TOO:     groupKeySet: {
-          [1650610759.578291][2739:2744] CHIP:TOO:       GroupKeySetID: 42
-          [1650610759.578334][2739:2744] CHIP:TOO:       GroupKeySecurityPolicy: 0
-          [1650610759.578363][2739:2744] CHIP:TOO:       EpochKey0: null
-          [1650610759.578391][2739:2744] CHIP:TOO:       EpochStartTime0: 2220000
-          [1650610759.578418][2739:2744] CHIP:TOO:       EpochKey1: null
-          [1650610759.578445][2739:2744] CHIP:TOO:       EpochStartTime1: 2220001
-          [1650610759.578472][2739:2744] CHIP:TOO:       EpochKey2: null
-          [1650610759.578498][2739:2744] CHIP:TOO:       EpochStartTime2: 2220002
-          [1650610759.578527][2739:2744] CHIP:TOO:      }
-          [1650610759.578553][2739:2744] CHIP:TOO:    }
-      disabled: true
-
-    - label:
-          "TH reads GroupKeyMap Attribute from the GroupKeyManagement cluster
-          from DUT"
-      verification: |
-          ./chip-tool groupkeymanagement read group-key-map 1 0
-
-          Verify  GroupKeyMapStruct on the TH Log:
-
-          1652429573.936605][2883:2888] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_003F Attribute 0x0000_0000 DataVersion: 75110822
-          [1652429573.936744][2883:2888] CHIP:TOO:   GroupKeyMap: 1 entries
-          [1652429573.936819][2883:2888] CHIP:TOO:     [1]: {
-          [1652429573.936860][2883:2888] CHIP:TOO:       GroupId: 1
-          [1652429573.936896][2883:2888] CHIP:TOO:       GroupKeySetID: 42
-          [1652429573.936932][2883:2888] CHIP:TOO:       FabricIndex: 1
-          [1652429573.936969][2883:2888] CHIP:TOO:      }
-      disabled: true
-
-    - label:
-          "TH reads GroupTable attribute from GroupKeyManagement cluster on DUT."
-      verification: |
-          ./chip-tool groupkeymanagement read group-table 1 0
-
-          Verify  GroupTable on the TH Log:
-
-          [1655965817.917300][3012:3017] CHIP:TOO: Endpoint: 0 Cluster: 0x0000_003F Attribute 0x0000_0001 DataVersion: 1208492408
-          [1655965817.917408][3012:3017] CHIP:TOO:   GroupTable: 1 entries
-          [1655965817.917506][3012:3017] CHIP:TOO:     [1]: {
-          [1655965817.917559][3012:3017] CHIP:TOO:       GroupId: 1
-          [1655965817.917600][3012:3017] CHIP:TOO:       Endpoints: 1 entries
-          [1655965817.917680][3012:3017] CHIP:TOO:         [1]: 1
-          [1655965817.917718][3012:3017] CHIP:TOO:       GroupName: grp1
-          [1655965817.917752][3012:3017] CHIP:TOO:       FabricIndex: 1
-          [1655965817.917784][3012:3017] CHIP:TOO:      }
-      disabled: true
-
-    - label:
-          "Verify each returned GroupInfoMapStruct contains an expected
-          GroupName as generated in step 1b."
+      command: "ViewGroup"
+      arguments:
+          values:
+              - name: GroupID
+                value: 0x0103
+      response:
+          values:
+              - name: Status
+                value: 0
+              - name: GroupId
```

Comment:
Would expect the name here and in similar places to be GroupID?

Reply:
Yeah, GroupID for view groups, GroupId in key management :) . Either way, the zapt does some fancy case changes.